test_util/test_upgrade_vpc.py

```diff
@@ -473,6 +590,21 @@ def apply(self, dcos_url: str, masters: List[str], public_masters: List[str], sl
                               default_os_user, DcosUser(CI_CREDENTIALS))
 
 
+class VpcClusterUpgradeTest(VpcClusterUpgradeTestFactory):
+    def mesos_metrics_snapshot(self, cluster, host):
```

Comment:
This is running in Ssher now, so its a pain to override this in a child of cluster because then you need to remake the cluster constructor. However, I don't think this should be made into a new class. The old class is fine and the downstream can just override...